pyproject.toml

```diff
@@ -24,10 +24,10 @@ classifiers = [
 [tool.poetry.dependencies]
 # pycollada is not working in Python 3.9. See
 # https://github.com/lace/tri-again/issues/25
-python = ">=3.7,<3.9"
```

Comment:
Before changing this version, could you address the above comment? It looks like pycollada needs to use 0.7.2 or greater. After changing that, this comment can be removed and when this PR is closed, we shoudl also close #25.